layouts/try/baseof.html

```diff
@@ -140,8 +140,10 @@ <h2>Interactive Tutorials</h1>
                 <div class="row">
                     <div class="col-md-4 pl-1 pr-1 pb-3">
                         <article class="card card-custom-style"
-                                 style="background-color: hsl(171, 100%, 41%); opacity: 0.4;"
-                                 onclick="">
+                                 style="background-color: hsl(171, 100%, 41%)"
+                                 data-katacoda-src="ncskiert/tekton-dashboard"
+                                 data-github-lnk="ncskier/katacoda/tree/master/pipelines"
```

Comment:
it looks like a couple things were missing:
* ncskier vs ncskiert
* pipelines vs tekton-dashboard in the github url
* missing the `interactive-tutorial-trigger` class